domains/Accounts/Controllers/VerifyEmailController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Domains\Accounts\Controllers;
+
+use App\Http\Controllers\Controller;
+use Domains\Accounts\Models\User;
+use Illuminate\Contracts\View\View;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Crypt;
+
+class VerifyEmailController extends Controller
+{
+    protected ?User $user = null;
+
+    public function __invoke(Request $request): View
+    {
+        $this->user = User::findOrFail($request->route('id'));
+        $hash = \base64_decode($request->route('hash'));
+
+        $this->authorizeForUser($this->user, 'verify', [User::class, $hash]);
```

Comment:
As discussed, let's revert this. It turned out not to be the enhancement we though it'll be.